From 1194c23407f04153e541d4dc636e1dfa83786624 Mon Sep 17 00:00:00 2001
From: Philip Withnall <philip.withnall@collabora.co.uk>
Date: Wed, 23 Mar 2016 08:58:43 +0000
Subject: main: Remove stray semicolon

This fixes a memory leak on exit.

Spotted by Coverity (CID: #78510).

https://bugs.freedesktop.org/show_bug.cgi?id=94669

diff --git a/src/gclue-main.c b/src/gclue-main.c
index 9872231..3e1416f 100644
--- a/src/gclue-main.c
+++ b/src/gclue-main.c
@@ -191,7 +191,7 @@ main (int argc, char **argv)
         main_loop = g_main_loop_new (NULL, FALSE);
         g_main_loop_run (main_loop);
 
-        if (manager != NULL);
+        if (manager != NULL)
                 g_object_unref (manager);
         g_bus_unown_name (owner_id);
         g_main_loop_unref (main_loop);
-- 
cgit v0.10.2

